projects/aca-content/src/lib/services/node-actions.service.ts

```diff
@@ -563,6 +563,47 @@ export class NodeActionsService {
     return null;
   }
 
+  getInfoDrawerIcon(node: Node) {
+    if (node.isFolder) {
+      return this.getFolderIcon(node);
+    }
+    if (node.isFile) {
+      return this.thumbnailService.getMimeTypeIcon(node.content.mimeType);
+    }
+    return this.thumbnailService.getDefaultMimeTypeIcon();
+  }
+
+  private getFolderIcon(node: Node) {
+    if (this.isSmartFolder(node)) {
+      return this.thumbnailService.getMimeTypeIcon('smartFolder');
+    } else if (this.isRuleFolder(node)) {
+      return this.thumbnailService.getMimeTypeIcon('ruleFolder');
+    } else if (this.isALinkFolder(node)) {
+      return this.thumbnailService.getMimeTypeIcon('linkFolder');
+    } else {
+      return this.thumbnailService.getMimeTypeIcon('folder');
+    }
+  }
+
+  isSmartFolder(node: Node) {
+    const nodeAspects = this.getNodeAspectNames(node);
+    return nodeAspects.indexOf('smf:customConfigSmartFolder') > -1 || nodeAspects.indexOf('smf:systemConfigSmartFolder') > -1;
```

Comment:
Use array.includes('') instead of checking the index

Reply:
Done